packages/dso-toolkit/src/components/tabs/tabs.scss

```diff
@@ -99,8 +99,9 @@
       display: block;
       line-height: typography.$line-height-base;
       margin-inline-end: 2px;
-      min-block-size: css-tabs-variables.$height;
-      padding: css-tabs-variables.$horizontal-padding css-tabs-variables.$vertical-padding;
+      min-block-size: css-tabs-variables.$block-size;
+      padding-block: css-tabs-variables.$inline-padding;
```

Comment:
Maar ik heb hier nog wel iets aan toe te voegen. 

Wanneer passen we het gebruik van variabelen toe? Ook als ze op 1 plek gebruikt worden? Ik vind dat in wezen overbodig. Pas wanneer een waarde op 2 of meer plekken in de zelfde context of betekenis wordt gebruikt lijkt mij het gebruik van een variabele meerwaarde hebben.

Dus in dit geval zou ik er ook prima mee kunnen leven dat de variabelen `$inline-padding` en `$block-padding` verdwijnen.

Reply:
Ik denk dat je dan @tfrijsewijk op je weg vindt; hij heeft een grote voorkeur voor 'semantische variabelen'. Ik opperde ook laatst om dat soort dingen te vervangen door bijv `units.$u2`, maar dat ging 'm niet worden ;)

Reply:
😁 

Ik hecht inderdaad veel waarde aan semantische variablen. En daarnaast passen semantische variablen ook beter op Design Tokens.

Dat een variable maar één keer gebruik wordt is geen afweging meer voor het bestaansrecht van een variable. Waar we het in de toekomst in moeten zoeken is de semantiek en wie de waarde beslist.

Als een beslissing door een designer is genomen, dan komt het voort uit een design token en dus een variable. Als een waarde door een developer wordt bepaald, bijvoorbeeld vanuit technische beperkingen (bv. bij shorthand CSS properties of tegenstijlen van default user agent styling) dan kan een developer de afweging maken of er een losse/semantische/extra sass variable nodig is.

Helpt dit betoog ergens iets?